llvm/test/Transforms/HotColdSplit/outline-inner-region-stacktoocomplex.ll

```diff
@@ -0,0 +1,51 @@
+; RUN: opt -S -passes=hotcoldsplit -hotcoldsplit-max-params=1 < %s | FileCheck %s
+
+target datalayout = "E-m:a-p:32:32-i64:64-n32"
+target triple = "powerpc64-ibm-aix7.2.0.0"
+
+define void @foo(i32 %cond) {
+; CHECK-LABEL: define {{.*}}@foo(
+; CHECK:       if.then:
+; CHECK:       if.then1:
+; CHECK:       if.else:
+; CHECK:       call {{.*}}@sink
+; CHECK:       if.end:
+; CHECK:       if.end2:
+;
```

Comment:
Probably better to use `llvm/utils/update_test_checks.py` to generate the full checks